src/ColorPicker/index.jsx

```diff
@@ -66,18 +66,20 @@ function ColorPicker({
           style={{ textAlign: 'start' }}
         >
           <HexColorPicker color={color || ''} onChange={setColor} />
-          <Form.Group className="pgn__hex-form" size="sm">
-            <h5>Hex</h5>
-            <Form.Control
-              className="form-field"
-              isInvalid={!hexValid}
-              value={color}
-              onChange={(e) => validateHex(e.target.value)}
-              data-testid="hex-input"
-            />
+          <Form.Group className="pgn__hex-form flex-wrap" size="sm">
+            <div className="d-inline-flex">
+              <h5>Hex</h5>
```

Comment:
The `Form.Control` below does not have a label for a11y. From a screen reader's POV, there is no connection between this "Hex" heading and the `Form.Control` input field.

If you can't make use of `Form.Label` here, might recommend implementing `aria-labelledby` to associate the `h5` with the `Form.Control`.

See this example from the W3: https://www.w3.org/WAI/tutorials/forms/labels/#using-aria-labelledby